### PR TITLE
Eng 1776 replace sidebars with new pages

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes
This PR replaces our current sidebar contents with the new details pages.

Still a little bit of layout cleanup to do in order to make pages more responsive, but I didn't want to touch that just yet in this PR in case we have other design things to address.

## Related issue number (if any)
ENG-1776

## Loom demo (if any)

NEW demo:
https://www.loom.com/share/25f1aaad77844e7ea639cab2687a6aab

old demo:
https://www.loom.com/share/f6be03c56f8d487bab6db66ca0bed0d5

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [x] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/pages/artifact/id/index.tsx [hunk 1]
```diff
@@ -32,14 +32,36 @@ import { LayoutProps } from '../../types';
...
+
+  if (workflowDagResultIdProp) {
+    workflowDagResultId = workflowDagResultIdProp;
+  }
+
+  if (operatorIdProp) {
+    artifactId = operatorIdProp;
+  }
+
+  console.log('workflowId: ', workflowId);
```
Comment (agiron123) [inline src/ui/common/src/components/pages/artifact/id/index.tsx:62]: remove logs

File: src/ui/common/src/components/pages/metric/id/index.tsx [hunk 2]
```diff
@@ -21,14 +21,40 @@ import { LayoutProps } from '../../types';
...
+
+  if (workflowDagResultIdProp) {
+    workflowDagResultId = workflowDagResultIdProp;
+  }
+
+  if (operatorIdProp) {
+    metricOperatorId = operatorIdProp;
+  }
+
+  console.log('workflowId: ', workflowId);
```
Comment (agiron123) [inline src/ui/common/src/components/pages/metric/id/index.tsx:55]: logs

File: src/ui/common/src/components/pages/workflow/id/index.tsx [hunk 3]
```diff
@@ -433,3 +438,5 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
 };
 
 export default WorkflowPage;
+
+//  let { workflowId, workflowDagResultId, operatorId } = useParams();
```
Comment (agiron123) [inline src/ui/common/src/components/pages/workflow/id/index.tsx:442]: remove commented out code.

File: src/ui/common/src/utils/sidesheets.tsx [hunk 4]
```diff
@@ -33,12 +47,49 @@ export function getDataSideSheetContent(
...
+          workflowDagResultIdProp={workflowDagResultIdProp}
+          workflowIdProp={workflowIdProp}
+        />
+      );
     case NodeType.ExtractOp:
     case NodeType.LoadOp:
-    case NodeType.FunctionOp:
-      return <OperatorResultsSideSheet user={user} currentNode={currentNode} />;
+    case NodeType.FunctionOp: {
+      //return <OperatorResultsSideSheet user={user} currentNode={currentNode} />;
```
Comment (agiron123) [inline src/ui/common/src/utils/sidesheets.tsx:83]: remove comment

**Review by likawind — APPROVED** [review]
This is very cool! My only comment for this PR is the left padding space seems a bit tight, and top is a bit too much?

Another thought beyond this PR -
Clicking on 'view artifact detail' now feel like expanding the sidebar to full page. It would be interesting if we can reverse this by shrinking from a detail page to a sidebar state

**Comment by agiron123** [comment]
> This is very cool! My only comment for this PR is the left padding space seems a bit tight, and top is a bit too much?
> 
> Another thought beyond this PR - Clicking on 'view artifact detail' now feel like expanding the sidebar to full page. It would be interesting if we can reverse this by shrinking from a detail page to a sidebar state

1. I'll add some more padding and see about adjusting the top padding too.

I would love to shrink from a detail page to a sidebar state, but that might be a little difficult to do at the moment. I need to look into it some more, but i'm not sure how nicely the drawer component (we're using default drawer component from MUI) plays with expanding / contracting it's size.

**Comment by agiron123** [comment]
@likawind fixed up things to your suggestions. New loom video can be found here:
https://www.loom.com/share/50ec1a680d4e40b896e68ddfb28b679c

**Pull request merged**